vendor/github.com/google/cadvisor/fs/fs.go

```diff
@@ -569,7 +569,7 @@ func GetDirDiskUsage(dir string, timeout time.Duration) (uint64, error) {
 	if dir == "" {
 		return 0, fmt.Errorf("invalid directory")
 	}
-	cmd := exec.Command("ionice", "-c3", "nice", "-n", "19", "du", "-s", dir)
+	cmd := exec.Command("ionice", "-c3", "nice", "-n", "19", "du", "-x", "-s", dir)
```

Comment:
unsure if we can patch vendor like this.  It will probably get overriden on the next vendor update?